index.js

```diff
@@ -28,7 +29,19 @@ const exampleMovies = require("./movies");
       "James and the Giant Peach",
     ];
  */
-function getAllMovieTitles() {}
+function getAllMovieTitles(movies) {
+  let movieTtiles = [];
+
+  if (movies.length === 0){
+    return movieTtiles;
+  }
+
+  for (let i = 0; i < movies.length; i ++){
+    movieTtiles.push(movies[i].title);
+  }
+
+  return movieTtiles;
+}
```

Comment:
Nice job here but let's remove the whitespace.  These breaks in logic are pretty easy to follow so I wouldn't recommend breaking them up.